cookbooks/apache2/kitchen.dokken.yml

```diff
@@ -1,69 +1,67 @@
----
 driver:
   name: dokken
-  # because Docker and SystemD
-  privileged: true
+  privileged: true  # because Docker and SystemD
   chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
-  env: [CHEF_LICENSE=accept]
 
 transport:
   name: dokken
 
 provisioner:
   name: dokken
+  chef_license: accept-no-persist
 
 platforms:
+  - name: almalinux-8
+    driver:
+      image: dokken/almalinux-8
+      pid_one_command: /usr/lib/systemd/systemd
+
   - name: amazonlinux-2
     driver:
       image: dokken/amazonlinux-2
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: debian-9
+  - name: debian-10
```

Comment:
why not go to debian-11?

Reply:
We don't use kitchen tests for our chef-third-party cookbooks! So, Can be ignored!